tester/src/test/scala/fixes/Issue963.scala

```diff
@@ -1,10 +1,10 @@
-package spinal.tester.scalatest
+package fixes
```

Comment:
This package name is a little bit weird.

Reply:
I initially put `issues` but then found a PR test which had no issue attached to it, so this is why I have chosen this name: to fix bugs.

Maybe something like `bugs`, `ghbugs`, `bughistory`?